content/use-cases/payments/restricting-deposits-uc.md

```diff
@@ -7,22 +7,21 @@ labels:
 ---
 # Restricting Deposits
 
-To comply with banking regulations, financial institutions must provide documentation about the sources of funds they receive. These regulations seek to prevent illicit activity by requiring institutions to track the source and destination of all payments they process. On the XRP Ledger, payments can be sent and received without any interaction from the receiver. This default behavior can be problematic, but you can enable deposite authorization so you only receive funds you explicitly approve.
+To comply with banking regulations, financial institutions must provide documentation about the sources of funds they receive. These regulations seek to prevent illicit activity by requiring institutions to track the source and destination of all payments they process. On the XRP Ledger, payments can be sent and received without any interaction from the receiver. This default behavior can be problematic, but you can enable deposit authorization so you only receive funds you explicitly approve.
 
 Accounts with deposit authorization enabled can only receive funds through:
 
   - Preauthorized Accounts
-  - Escrow
-  - Payment Channels
   - Checks
+  - Escrow
+<!-- - Payment Channels -->
```

Comment:
Aren't payment channels also a valid way of receiving funds?

Reply:
The section on payment channels is commented out. If we're not ready to discuss Payment Channels in this contest, I think it's better to omit the bullet point.

Reply:
Makes sense to me. No other questions!